src/porepy/numerics/fv/mpfa.py

```diff
@@ -52,14 +51,12 @@ def discretize(self, g, data):
                 the permeability (such as with fracture apertures) should be
                 included in the permeability.
             bc: (BoundaryCondition) boundary conditions
-            vector_source: (boolean) Optional. Discretization of the divergence of
-                a vector source. For example, gravity in the flow equations. If False,
-                standard MPFA is used.
+            vector_source_dim: (int) Optional. Ambient dimension, used in the
```

Comment:
I didn't see this in tpfa

Reply:
You are right. I added the tpfa functionality towards the end, but have not fixed embedded grids. That also shows the test coverage is not good enough.